iac-extensions/arm/src/main/java/org/sonar/iac/arm/tree/api/bicep/Decorator.java

```diff
@@ -20,8 +20,9 @@
 package org.sonar.iac.arm.tree.api.bicep;
 
 import org.sonar.iac.arm.tree.api.ArmTree;
+import org.sonar.iac.arm.tree.api.Expression;
 
 public interface Decorator extends ArmTree {
 
-  FunctionCall functionCall();
+  Expression expression();
```

Comment:
I think for later when we'll need to analyze decorators and retrieve some values, it would be useful to have an interface `DecoratorExpression` with an accessor for function call

Reply:
As we are not sure if rules would really use it, for now I would leave it as it is. Can be later added when we have Checks than would benefit from it IMO